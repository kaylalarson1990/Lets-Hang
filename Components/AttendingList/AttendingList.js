import React from 'react'
import { View, Text } from 'react-native'


export const AttendingList = (props) => {
  return (
    <View>

    </View>
  )
}