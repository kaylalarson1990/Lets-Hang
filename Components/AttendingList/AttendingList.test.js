import 'react-native'
import React from 'react'
import { AttendingList } from './AttendingList'

import { shallow } from 'enzyme'

describe('AttendingList', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<AttendingList name={'ryan'} />)
  })
  it('Should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})