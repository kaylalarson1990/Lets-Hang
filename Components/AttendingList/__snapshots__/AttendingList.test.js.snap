// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AttendingList Should match the snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
>
  <Text
    style={
      Object {
        "margin": 2,
      }
    }
  >
    ryan
  </Text>
</TouchableOpacity>
`;
